markup/house-style.md

```diff
@@ -80,9 +82,11 @@ We *don't* do this:
 
 When HTML 5 was first announced, it brought with it a plethora of new semantic elements and an entirely new document outline model which leveraged them to provide meaning.  This was intended to supplant the HTML 4 model in which the heading level (h1 - h6) was used to imply the structure of the HTML document.  See [Using HTML sections and outlines](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines) for more information.
 
-Unfortunately, this new document outline model was never well supported by browsers or assistive technologies.  We considered sticking with the HTML 5 document outline model used in conjunction with setting ARIA attributes (`role="heading" aria-level="3"`) but this rather defeated any benefit from using the more localised heading levels, and was also poorly supported in some key assistive technologies.
+Unfortunately, this new document outline model was never well supported by browsers or assistive technologies. We considered sticking with the HTML 5 document outline model used in conjunction with setting ARIA attributes (`role="heading" aria-level="3"`) but this rather defeated any benefit from using the more localised heading levels, and was also poorly supported in some key assistive technologies.
+
+The HTML 5 document outline algorithm specification has been removed in HTML 5.1. You should continue to use the older HTML 4 style heading levels in conjunction with the new HTML 5 semantic elements. Do not use multiple H1 headings on a single page. 
```

Comment:
Are multiple `H1` headings disallowed by the spec or our best practise?

Reply:
I know I'd personally prefer to use a single H1 and branch all headings from there. But I don't know what our gestalt view is.

Reply:
Our best practice. We need to interpret the specification and the WAI guidelines and decide what's best based on our interpretation. 

* The HTML spec says "[the h1 is the top level heading](https://www.w3.org/TR/html5/sections.html#the-h1-h2-h3-h4-h5-and-h6-elements)" 
* The WAI recommendation says "" https://www.w3.org/WAI/tutorials/page-structure/headings/

Neither of them _explicitly_ rule out having multiple H1s on the page using those exact words. Semantically I'd argue that a single document has a single purpose, and the H1 reflects that purpose. 

That said, the aXe automated accessibility testing ruleset doesn't permit more than one H1 per page, so if we want to pass automated accessibility checking, then we'll have to adhere to that rule: https://dequeuniversity.com/rules/axe/3.0/page-has-heading-one

We also know that 60% of screen reader users responding to WebAIM's screen reader survey prefer a single H1 containing the document title: https://webaim.org/projects/screenreadersurvey7/#heading

Reply:
I checked the HTML_Codesniffer rules, and they permit multiple h1s. Pa11y currently only uses HTML_CS, as the work to integrate axe is still pending, but I'm inclined to go with axe's judgement on this

Reply:
> - The HTML spec says "the h1 is the top level heading"
> - The WAI recommendation says "The most important heading has the rank 1 (h1)"
> - 60% of screen reader users responding to WebAIM's screen reader survey prefer a single H1 containing the document title

I think those make the case just by themselves.

Reply:
> Semantically I'd argue that a single document has a single purpose, and the H1 reflects that purpose.

I agree.

Reply:
I agree. Before I wrap it up though, there's a really interesting - lengthy - discussion about this on the WebAIM email archive: https://webaim.org/discussion/mail_thread?thread=6343

I find the argument advanced by Jared Smith (Mon, Mar 10 2014 2:59PM) to be compelling (largely because it's in line with my existing confirmation bias r.e. semantic choices, but hey ho). Search for the date line to find it if you're interested - he talks a bit about how some screen reader users navigate by heading, and also about the semantic implications of ranking document titles and subsections. It's a great comment, really interesting.